### ISSUE TITLE
Button to join session is jumpy and unclickable (latest Chrome)

### ISSUE THREAD

**Issue by marclaporte** [mozilla/togetherjs#1023]
Check the video here: https://dev.tiki.org/item5434

TogetherJS is integrated in Tiki Wiki CMS Groupware 12.x:
http://doc.tiki.org/PluginTogether

Thanks!

**Comment by marclaporte** [comment]
That was an old report from 12 years ago so makes sense to close.

I am still curious and interested about this project.

**Comment by oskarkrawczyk** [comment]
Yeah, this project has been transferred from Mozilla to us a long time ago. 

I've pushed some updates today for the modern era (like the ability to run the hub server for free on Cloudflare Workers).

As to the old issues and pull requests - I have to be honest with myself, not going to bother with these.
